src/Discord.Net.Interactions/Builders/ModuleClassBuilder.cs

```diff
@@ -18,7 +18,7 @@ public static async Task<IEnumerable<TypeInfo>> SearchAsync(Assembly assembly, I
         {
             static bool IsLoadableModule(TypeInfo info)
             {
-                return info.DeclaredMethods.Any(x => x.GetCustomAttribute<SlashCommandAttribute>() != null);
+                return !info.IsAbstract && info.DeclaredMethods.Any(x => x.GetCustomAttribute<SlashCommandAttribute>() != null);
```

Comment:
I think doing something like this for the attribute check would fix the problem I was mentioning.
```suggestion
               return !info.IsAbstract && info.DeclaredMethods.SelectMany(x => x.GetCustomAttributes()).Any(x => x switch
    {
        SlashCommandAttribute or ComponentInteractionAttribute or ContextCommandAttribute or AutocompleteCommandAttribute or ModalInteractionAttribute => true,
        _ => false
    });
```
I tested this but can you please also test it so we don't run into any issues later.

Also the only other part as far as I can see is on #L61, it should say interaction module instead of slash command module. Other than that the PR should be good to go.